storage/rocksdb/ha_rocksdb.cc

```diff
@@ -13081,13 +13081,29 @@ static int rocksdb_validate_update_cf_options(
     my_error(ER_WRONG_VALUE_FOR_VAR, MYF(0), "rocksdb_update_cf_options", str);
     return HA_EXIT_FAILURE;
   }
-  // Loop through option_map and create missing column families
+  // Loop through option_map and check if all specified CFs exist.
+  std::vector<std::string> unknown_cfs;
   for (Rdb_cf_options::Name_to_config_t::iterator it = option_map.begin();
        it != option_map.end(); ++it) {
-    if (!cf_manager.get_or_create_cf(rdb, it->first,
-                                     !rocksdb_no_create_column_family)) {
-      return HA_EXIT_FAILURE;
+    if (!cf_manager.get_cf(it->first)) {
+      unknown_cfs.push_back(it->first);
+    }
+  }
+
+  if (unknown_cfs.size() > 0) {
+    std::string err(str);
+    err.append(" Unknown CF: ");
+    bool first = true;
+    for (const auto cf : unknown_cfs) {
```

Comment:
I'm not sure it it will make generation of error message easier to read. We need to deal with trailing coma.
ostringstream can be also used to construct the final string, but again, I'm not sure if it is worth introducing dependency to ostringstream (other parts in this file use string concatenation by string::append())